lib/public_xml_service.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+class PublicXmlService
+  attr_reader :object
+
+  def initialize(object)
+    @object = object
+  end
+
+  def to_xml
+    pub = Nokogiri::XML('<publicObject/>').root
+    pub['id'] = object.pid
+    pub['published'] = Time.now.utc.xmlschema
+    pub['publishVersion'] = 'dor-services/' + Dor::VERSION
```

Comment:
:eyes: do we care about this lying or meh?

Reply:
Feel free to merge if meh.

Reply:
I honestly have no clue as to what the consumer expects, so I don't want to change anything.